cjsmin/src/chess.ts

```diff
@@ -1458,19 +1514,30 @@ export class Chess {
         } else {
           index = move.to + 16;
         }
-        throw new Error('TODO: Implement en passant capture');
+        // determine captured pawn based on square of captured piece
+        const square = algebraic(move.to);
+        let unambiguousSymbol = '';
+        if (square.charAt(1) === '2') {
+          unambiguousSymbol += 'P';
```

Comment:
good catch on setting white pawns to be capitalized and black to be lowercased, thx

Reply:
I actually think this isn't necessary, reverting it to just use capture.UAS